### DISCUSSION TITLE
Workflow dispatch

### DISCUSSION THREAD

**Discussion by itsmesuniljacob** [dflook/terraform-github-actions#89] [Q&A] [answered]
@dflook 
 Does the apply action run on workflow_dispatch?

**Comment by dflook** [comment, marked as answer]
The apply action can be triggered by workflow_dispatch if `auto_approve: true`.
workflow_dispatch doesn't relate to a PR, so there is nowhere to pull a plan from.

**Reply by opsline-jvarelas** [reply]
hey @dflook what if I'm passing in the inputs the plan_path, should the terraform-apply takes the plan instead of trying to generate or grab it from the PR?

**Reply by dflook** [reply]
Yes, when setting `auto_approve: true` and `plan_path` the plan file is applied directly. No plan is generated and no PR comment is used.

**Reply by opsline-jvarelas** [reply]
hey @dflook first of all thanks for your amazing work, those Gh actions are very helpful, 
I have this scenario:
* I created a GH workflow to be triggered manually (workflow_dispatch)
* Im using the terraform-plan and if there're resources to destroy we have a manual approval
* Im passing the plan.tfplan to the terraform-apply to avoid a new plan, but since im not using a PR its failing, 

It'd be great if we can use the input **plan_path**, even if we dont have a PR

thanks in advance.

**Reply by dflook** [reply]
@opsline-jvarelas Please create an issue to describe the problem
